modules/Notes/Note.php

```diff
@@ -205,6 +205,21 @@ public function get_summary_text()
         return "$this->name";
     }
 
+    /**
+     * Returns the content as string or false if there is no attachment or it
+     * couldn't be located.
+     *
+     * @return bool|string
+     */
+    public function getAttachmentContent()
+    {
+        $path = "upload://{$this->id}";
+        if (!file_exists($path))
```

Comment:
Please use { } with if statements.

For-example: 

```
if (!file_exists($path)) {
    return false;
}
```

Reply:
thanks, done (also in other places)